habitat-lab/habitat/tasks/rearrange/rearrange_task.py

```diff
@@ -243,8 +243,11 @@ def reset(self, episode: Episode, fetch_observations: bool = True):
             self._is_episode_active = True
 
             if self._should_place_articulated_agent:
+                # here we are randomizing the base pos and rotation if necessary
```

Comment:
That config field name is ambiguous. It may be worth the trouble renaming (not in this PR).

Reply:
Yeah, this part of the code is ripe for refactor